src/_zkapauthorizer/resource.py

```diff
@@ -256,23 +257,59 @@ def render_POST(self, request):
             request.setResponseCode(BAD_REQUEST)
             return b"recovery-capability must be a read-only dircap string"
 
+        # The response to this request does not wait for recovery to complete.
+        # Instead, a separate endpoint exposes the progress of the recovery
+        # attempt.  So we're not going to wait for `recovering` to complete.
+        # However, we do have to schedule it in the event loop or it will
+        # never even start.
+        recovering = self._recover(request, self.store, cap)
+        Deferred.fromCoroutine(recovering)
```

Comment:
We're just dropping this `Deferred` .. should it at least have an `addErrback` that does something appropriate? (e.g. at least log the error?) I guess ideally it should tell "something" that the recovery is now never going to happen.